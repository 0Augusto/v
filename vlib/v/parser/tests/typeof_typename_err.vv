fn f[T]() { println("type ${typeof(T)}") }
f[int]()
f[string]()

		  
