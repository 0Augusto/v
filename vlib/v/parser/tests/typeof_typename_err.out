vlib/v/parser/tests/typeof_typename_err.vv:1:29: error: use `typeof[T]()`, instead of `typeof(T)`
    1 | fn f[T]() { println("type ${typeof(T)}") }
      |                             ~~~~~~
    2 | f[int]()
    3 | f[string]()
